src/_gettsim/parameters/erwerbsm_rente.yaml

```
@@ -0,0 +1,256 @@
+    scalar: 62.416666
+  2001-08-01:
+    scalar: 62.333333
+  2001-09-01:
+    scalar: 62.25
+  2001-10-01:
+    scalar: 62.166666
+  2001-11-01:
+    scalar: 62.083333
+  2001-12-01:
+    scalar: 62
+  2002-01-01:
+    scalar: 61.916666
+  2002-02-01:
+    scalar: 61.833333
+  2002-03-01:
+    scalar: 61.75
+  2002-04-01:
+    scalar: 61.666666
+  2002-05-01:
+    scalar: 61.583333
+  2002-06-01:
+    scalar: 61.5
+  2002-07-01:
+    scalar: 61.416666
+  2002-08-01:
+    scalar: 61.333333
+  2002-09-01:
+    scalar: 61.25
+  2002-10-01:
+    scalar: 61.166666
+  2002-11-01:
+    scalar: 61.083333
+  2002-12-01:
+    scalar: 61
+  2003-01-01:
+    scalar: 60.916666
+  2003-02-01:
+    scalar: 60.833333
+  2003-03-01:
+    scalar: 60.75
+  2003-04-01:
+    scalar: 60.666666
+  2003-05-01:
+    scalar: 60.583333
+  2003-06-01:
+    scalar: 60.5
+  2003-07-01:
+    scalar: 60.416666
+  2003-08-01:
+    scalar: 60.333333
+  2003-09-01:
+    scalar: 60.25
+  2003-10-01:
+    scalar: 60.166666
+  2003-11-01:
+    scalar: 60.083333
+  2003-12-01:
+    scalar: 60
+  2014-07-01:
+    scalar: 62
+  2019-01-01:
+    scalar: 65.666666
+  2020-01-01:
+    scalar: 65.75
+  2021-01-01:
+    scalar: 65.833333
+  2022-01-01:
+    scalar: 65.916666
+  2023-01-01:
+    scalar: 66
+  2024-01-01:
+    scalar: 66.083333
+  2025-01-01:
+    scalar: 66.166666
+  2026-01-01:
+    scalar: 66.25
+  2027-01-01:
+    scalar: 66.333333
+  2028-01-01:
+    scalar: 66.5
+  2029-01-01:
+    scalar: 66.666666
+  2030-01-01:
+    scalar: 66.833333
+  2031-01-01:
+    scalar: 67
+min_zugangsfaktor:
+  name:
+    de: Kleinster möglicher Zugangsfaktor bei der Erwerbsminderungsrente
+    en: Smallest possible access factor for public disability insurance
+  description:
+    de: >-
+      Abzüge wegen frühzeitigem Eintritt in die Erwerbsminderungsrente
+      sind bei 10.8% gedeckelt.
+    en: Deductions for early retirement are capped at 10.8%.
+  unit: Factor
+  reference: § 77 SGB VI Zugangsfaktor
+  note: >-
+    Rentenreformgesetz 1999 (RRG 1999)
+    "http://www.portal-sozialpolitik.de/index.php?page=rentenversicherung"
+  2000-01-01:
+    scalar: 0.892
+rentenartfaktor:
+  name:
+    de: Rentenartfaktor für teilweise Erwerbsminderungsrente
+    en: Pension type factor for partial reduction in public disability insurance
+  description:
+    de: >-
+      Der Rentenartfaktor ist ein festgelegter Faktor für die
+      Rentenberechnung und  bestimmt
+      das Sicherungsziel der Rentenart im Verhältnis zu einer Altersrente.
+    en: >-
+      The pension type factor is a fixed factor for the pension calculation
+      and  determines
+      the security target of the pension type in relation to an old-age pension.
+  unit: Factor
+  reference: >-
+    § 67 SGB VI Rentenartfaktor:
+  note: >-
+    "https://sozialversicherung-kompetent.de/rentenversicherung/
+    leistungsrecht/1051-rentenartfaktor.html"
+  2001-01-01:
+    teilw: 0.5
+    voll: 1.0
+altersgrenze_grundbew:
```

```suggestion
altersgrenze_grundbewertung:
```
No reason for abbreviations here.